### PR TITLE
Updates Color Palette 🎨

### LINKED ISSUE #733: Change UI Text Colours
Let's change the UI text colours to more closely match the contrast that can be found in macOS system colours:

### Editor

**This includes**
- Title
- Body

**Colours**
- Light: Gray70 (#3c434a)
- Dark: Gray5 (#dcdcde)

 ### Navigation UI

**This includes**
- Tags list + icons
- Selected tag label (shown above the notes list eg. "All Notes")
- Tags within the editor

**Colours**
- Light: Gray90 (#1d2327)
- Dark: Gray5 (#dcdcde)

### Notes List

**This includes**
- Notes list rows titles and body text

**Colours**
Title:
- Light: Gray90 (#1d2327)
- Dark: Gray5 (#dcdcde)

Body:
- Light: Gray50 (#646970)
- Dark: Gray30 (#8c8f94)

### Search Bar

**This includes**
- Search bar placeholder
- Search bar input text

**Colours**
Title:
- Light: Gray50 (#646970)
- Dark: Gray30 (#8c8f94)

Body:
- Light: Gray90 (#1d2327)
- Dark: Gray5 (#dcdcde)

### Other

Change published icon (in the notes list) to:
- Light: Gray50 (#646970)
- Dark: Gray30 (#8c8f94)

Change "TAGS" label in the tags list to:
- Light: Gray50 (#646970)
- Dark: Gray30 (#8c8f94)

### PR DESCRIPTION
**Pull request by jleandroperez**
### Fix
In this PR we're (slightly!) adjusting the color palette.

@eshurakov New day, new UX!! Thanks in advance!!

Closes #733
Ref. #718

### Test
- [x] Please run a smoke test and make sure Light / Dark mode look stunning!

### Release
`RELEASE-NOTES.txt` was updated in 49b3276 with:

> Updated Color Palette #733

### Screenshots

Old Darth|New Darth
-|-
Old Darth|New Darth

Old Light|New Light
-|-
Old Light|New Light

### CODE REVIEW

**Comment by SylvesterWilmott** [comment]
>Do we also wanna update those?

Yes, can we update those too please?

Looking good so far!

File: Simplenote/NoteListViewController.m [hunk 1]
```diff
@@ -84,11 +84,7 @@ - (void)viewDidLoad
     [self setupTableView];
     [self startListeningToScrollNotifications];
     [self startListeningToWindowNotifications];
-}
 
-- (void)viewWillAppear
-{
-    [super viewWillAppear];
```
Comment (jleandroperez) [inline Simplenote/NoteListViewController.m]: I'm not entirely sure how this change slipped in here (It's something I've been working on in another branch!).

Please disregard this one, **should** hopefully be inoccuous!!

**Review by SylvesterWilmott — APPROVED** [review]
Thanks! Looks great

**Review by eshurakov — APPROVED** [review]
Looks good to me 👍 

:shipit:

**Comment by jleandroperez** [comment]
Thank you @eshurakov + @SylvesterWilmott !!! 🔥

**Pull request merged**